pkg/kor/hpas.go

```diff
@@ -55,33 +55,33 @@ func processNamespaceHpas(clientset kubernetes.Interface, namespace string, filt
 		return nil, err
 	}
 
-	var unusedHpas []string
+	var unusedHpas []ResourceInfo
 	for _, hpa := range hpas.Items {
 		if pass, _ := filter.SetObject(&hpa).Run(filterOpts); pass {
 			continue
 		}
 
 		if hpa.Labels["kor/used"] == "false" {
-			unusedHpas = append(unusedHpas, hpa.Name)
+			unusedHpas = append(unusedHpas, ResourceInfo{Name: hpa.Name, Reason: "Marked as unused by label"})
```

Comment:
```suggestion
			unusedHpas = append(unusedHpas, ResourceInfo{Name: hpa.Name, Reason: "Marked with unused label"})
```
Alignment to reason in all resources.